dynatrace-aws-logs.sh

```diff
@@ -78,13 +81,16 @@ arguments:
                         Enables checking SSL certificate of the target Active Gate. By default (if this option is not provided) certificates aren't validated
   --stack-name STACK_NAME
                         Optional. The name for the CloudFormation stack in which the resources will be deployed. This defaults to \"$DEFAULT_STACK_NAME\"
+  --max-log-length MAX_LOG_CONTENT_LENGTH
+                        Optional, defaults to 8192. Defines the max log length after which a log will be truncated.
+                        Requires a cluster side change for values over 8192 (Manage -> Settings -> Internal -> Indexed Log Storage Settings -> Event content maximum byte size).
```

Comment:
I am not sure "Settings \ Internal" is accessible for people. I think you have to contact Dynatrace ONE for this change. Please double check, ask logs team

Reply:
@lukasz-parulski-dynatrace What about this one?

Reply:
It's corrected already :D
![image](https://user-images.githubusercontent.com/108682852/183449892-83fe45e0-2ebd-4559-8f07-6b1446b1d7b9.png)